docs/system-administration/ubiquiti.md

```diff
@@ -0,0 +1,18 @@
+# Ubiquiti
+
+This page contains information relating to the Ubiquiti line of products.
+
+## Adopt a new Access Point with remote Unifi controller software
+
+If your Unifi controller software doesn't run on your LAN it is neccessary to tell the AP where it is. This allows the AP to be 'adopted' by the controller and have its configuration managed by it.
+
+To do this you will need to SSH to the AP (default user and password is `ubnt`).
+
+    ssh <ip-of-access-point>
+    set-inform http://<controller-url>:8080/inform
+
+Once you've done this, the AP should show up almost immediately in the interface of your Unifi controller saying 'pending adoption'.
+
+Adopt the AP and configuration for that site will be automatically applied.
+
+> Note that after adoption the SSH password will be changed from ubnt/ubnt. You can modify the post adoption Unifi SSH username and password in the controller software by enabling 'advanced features' and then configuring your desired credentials under `Site -> Device Authentication`.
```

Comment:
I'd be interested in trying a proper note here

https://squidfunk.github.io/mkdocs-material/extensions/admonition/

Reply:
Will that work with hugo if / when we migrate? I've not come across this note thing before. I worry it's going to create technical debt?

Reply:
I don't believe that hugo will support this. Hugo generally uses short code to extend functionality. I'm not sure that that should stop us however If we want to really get a good idea of mkdocs then we should use it to its fullest.

Reply:
These can be considered similar to Hugo's shortcodes. I think it's unrealistic to expect a port to Hugo be free in terms of content modifications, and as @BCNelson says, we shoud use `mkdocs` to it's fullest, as we aren't necessarily migrating.